### PR TITLE
Ansible RBAC

### PR DESCRIPTION
**Pull request by dhaiducek**
Addresses https://github.com/open-cluster-management/backlog/issues/13823

Depends on API PR https://github.com/open-cluster-management/grc-ui-api/pull/148

### CODE REVIEW

File: src-web/components/modals/AnsibleAutomationModal/AnsibleAutomationModal.js [hunk 1]
```diff
@@ -555,27 +563,27 @@ export class AnsibleAutomationModal extends React.Component {
...
+            token,
           }
         })
       }
     }
-    return {
-      towerURL: targetCredential.host,
-      token: targetCredential.token
-    }
+    return { towerURL, token }
```
Comment (ChunxiAlexLuo) [inline src-web/components/modals/AnsibleAutomationModal/AnsibleAutomationModal.js:586]: Go through the whole logic and looks good to me. Thanks for fixing this line and other corner cases handling

**Review by ChunxiAlexLuo — APPROVED** [review]
/lgtm

**Review by ChunxiAlexLuo — APPROVED** [review]
/lgtm

**Review by willkutler — APPROVED** [review]

**Pull request merged**